include/galsim/SBInterpolatedImageImpl.h

```diff
@@ -26,9 +26,58 @@
 
 namespace galsim {
 
-    class SBInterpolatedImage::SBInterpolatedImageImpl : public SBProfile::SBProfileImpl
+    class SBInterpolated::SBInterpolatedImpl : public SBProfile::SBProfileImpl
     {
     public:
+
+        SBInterpolatedImpl(boost::shared_ptr<Interpolant2d> kInterp,
+                           double stepk, double maxk,
+                           const GSParamsPtr& gsparams);
+        ~SBInterpolatedImpl();
+
+        boost::shared_ptr<Interpolant> getKInterp() const;
+
+        bool isAxisymmetric() const { return false; }
+
+        // We'll use false here, but really, there's not an easy way to tell.
+        // Certainly an Image _could_ have hard edges.
+        bool hasHardEdges() const { return false; }
+
+        // This class will be set up so that both x and k domain values
+        // are found by interpolation of a table:
+        bool isAnalyticX() const { return true; }
```

Comment:
This should be false for `SBInterpolatedKImage`.  At least until we implement `xValue`.  That's really what this (poorly-named) function is about.  If this is true, then `xValue` is implemented.  A better name for these would be something like `hasDirectlyCalculableXValues`.